lib/Containers/Helpers.h

```diff
@@ -33,28 +33,29 @@ namespace arangodb::containers {
 struct Helpers {
 /// @brief calculate capacity for the container for at least one more 
 /// element.
-/// if this would exceed the container's capacity, use a factor-of-2
-/// growth strategy to calculate the capacity.
+/// if this would exceed the container's capacity, use a growth factor of
+/// 1.5 to calculate the new capacity.
 template<typename T>
 static std::size_t nextCapacity(T const& container, std::size_t initialCapacity) {
   std::size_t capacity;
   if (container.empty()) {
     // reserve some initial space
     capacity = std::max(std::size_t(1), initialCapacity);
   } else {
+    // minimum is that we have room for at least one more element.
     capacity = container.size() + 1;
-    // allocate with power of 2 growth
     if (capacity > container.capacity()) {
-      capacity *= 2;
+      // grow with a growth factor of 1.5
+      capacity = (container.size() * 3) / 2;
```

Comment:
I think there's a bug here if `container.size() == 1`, as it would set capacity to one.
```suggestion
      capacity = ((container.size() - 1) * 3) / 2 + 1;
```
As `!container().empty()`, this won't underflow. And we get capacity rounded up rather than down. Or we just use
```suggestion
      capacity = (container.size() * 3 + 1) / 2;
```
.

Reply:
Ok, I have applied the suggestion.